account_move_base_import/parser/generic_file_parser.py

```diff
@@ -0,0 +1,78 @@
+# © 2011 Akretion
+# © 2011-2016 Camptocamp SA
```

Comment:
```suggestion
# © 2011-2019 Camptocamp SA
```

Reply:
and everywhere

Reply:
I wonder if we can not just put `2011` instead of a year-range. I think it implies this year until today.